contracts/oracles/ChainlinkOracle.sol

```diff
@@ -123,12 +127,36 @@ contract ChainlinkOracle is IOracle {
             uint256
         )
     {
-        (uint80 round, int256 latestPrice, , uint256 latestTS, ) = aggregator.getRoundData(_round);
+        (uint80 round, int256 latestPrice, , uint256 latestTS, ) = _getRoundDataWithCheck(_round);
         while (latestPrice < 0 && round > 0) {
```

Comment:
should it be `latestPrice < 0` or `latestPrice <= 0` since default value of int256 is 0

Reply:
No, we want to stop when we get latestPrice = 0. Hence "latestPrice<0" is correct